Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN pip install --upgrade pip && pip install wheel
 RUN pip wheel . --wheel-dir=/wheels
 
 # Install from wheels
-FROM apeworx/ape:stable
+FROM ghcr.io/apeworx/ape:latest-slim
```

Comment:
Would like it to use stable, but needs a release for that

Reply:
Or maybe make it configurable?

Reply:
I can make it configurable for sure, I was going to as you if you wanted an ARG in there